chapters/equations.tex

```diff
@@ -555,7 +555,7 @@ \section{Synchronous Data-flow Principle and Single Assignment Rule}\label{synch
 Modelica is based on the synchronous data flow principle and the single
 assignment rule, which are defined in the following way:
 \begin{enumerate}
-\item Discrete variables keep their actual values until these variables are explicitly changed.
+\item Discrete-time variables keep their actual values until these variables are explicitly changed.
```

Comment:
By the way, the use of _actual_ here seems redundant — what are we trying to say, if anything?

Reply:
OK, then I suggest that we simply remove _actual_.

Reply:
Ok, done.